Assignment1/README.md

```diff
@@ -0,0 +1,75 @@
+# Nuxt 3 Minimal Starter
```

Comment:
Make sure to add your link to this README. I think it was in the requirements :)

Reply:
Thanks!